task3/task3/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace task3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("x=");
+            double x = Convert.ToDouble(Console.ReadLine());
+            double y = Math.Abs(2 * Math.Sin(-3 * Math.Abs((x + 1) / 2)));
+            Console.WriteLine("y={0}", y);
```

Comment:
Более современный подход - использование [интерполяции строк](https://docs.microsoft.com/ru-ru/dotnet/csharp/tutorials/exploration/interpolated-strings)
```suggestion
            Console.WriteLine($"y = {y}");
```